### PR TITLE
Milestone 2 - Added two more sections

### PR DESCRIPTION
**Pull request by fahim-86**
This is second milestone of the Portfolio mobile view page project. In this project two more section has been added. Those sections are:

- Work
- About

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -43,6 +43,138 @@ <h1>I’m Fahim Glad to see you!</h1>
...
+        <div class="pf-portfolio">
+          <div class="card">
+            <img src="./img/tonic.png" alt="Tonic section" />
+            <h2>Tonic</h2>
+            <div class="under-header-text">
+              <p>CANOPY</p>
+              <ul>
+                <li>Back End Dev</li>
+                <li>2015</li>
+              </ul>
```
Comment (brainconnect93) [inline index.html:56]: - [ ] I would kindly suggest you add the project description to each, as this talks about what each of the projects in the card entails. Currently, the details are missing from all your cards, by implementing this you have followed the Figma template as much as possible.

Your design 👇 

farim

Figma design 👇 

card real

File: styles.css [hunk 2]
```diff
@@ -14,15 +15,29 @@ h1 {
...
+
 .pf-wrap {
   max-width: 375px;
-  min-height: 718px;
+  height: auto;
   margin: 5vh auto 5vh auto;
-  background-image: url("../portfolio-website/img/header-background-mobile.png");
+  background-image: url("./img/header-background-mobile.png");
+  background-repeat: no-repeat;
+  background-color: #e5e5e5;
```
Comment (brainconnect93) [inline styles.css]: - [ ] I notice a grayish color at the top and bottom of your project, which is not supposed to be like that. I would suggest you change the CSS property in line 37 from `margin: 5vh auto 5vh auto;` to `margin: 0 auto;`. With this you will achieve removing the extra color showing at the top and bottom of your project

Your project 👇 

far1

**Review by brainconnect93 — CHANGES_REQUESTED** [review]
# STATUS: Changes Required :recycle:

### Hi @fahim-86, 👋 

Good job so far, You really made a great effort working on your project, it looks really awesome! but there are some issues that you still need to work on to go to the next project but you are almost there!



### To Highlight :1st_place_medal: 
-  Has a proper Pull Request. ✔️  
- No Linter errors :heavy_check_mark: 
- Good use of Github flow ✔️ 

### Changes Required

- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @brainconnect93 ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by KanzaTahreem — APPROVED** [review]
# Approved 🥳

Hi @fahim-86 

Well done with implementing the changes suggested by previous reviewer and nailing the project 💪

**_Your project is COMPLETE! There is nothing else to say other than... it's time to merge it! 🚀 Congratulations!_**



## Highlights 💡
- ✅ Great work using the right GitHub Flow 
- ✅ Commits are descriptive, good work.
- ✅ Linters are working properly, great job! 

## Optional suggestions  ❕❕
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

 - [OPTIONAL] I think it would be better if you add all headers/section of README as according to the [template provided](https://github.com/microverseinc/readme-template) to you. Currently, your README lacks few headings
 
### _**Your README 🔽**_
image

### _**README template 🔽**_
image


Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**